config/sync/migrate_plus.migration.cs_spotlight_migration.yml

```diff
@@ -0,0 +1,77 @@
+uuid: e945234d-6d9e-44d5-a5df-e49790460ce4
+langcode: en
+status: true
+dependencies: {  }
+id: cs_spotlight_migration
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags:
+  - JSON
+  - Spotlight
+migration_group: custom_content
+label: 'Import Spotlight from JSON'
+source:
+  plugin: url
+  data_fetcher_plugin: http
+  data_parser_plugin: json
+  urls: 'https://cardinalservice-dev.sites-pro.stanford.edu/spotlight-exporter'
+  item_selector: 0
+  constants:
+    basic_html: stanford_html
+    minimal_html: stanford_minimal_html
+  fields:
+    -
+      name: title
+      label: Title
+      selector: title
+    -
+      name: body
+      label: 'Body Content'
+      selector: body
+    -
+      name: su_spotlight_student_name
+      label: 'Spotlight student name'
+      selector: su_spotlight_student_name
+  ids:
+    title:
+      type: string
+process:
+  type:
+    plugin: default_value
+    default_value: stanford_news
+  title: title
+  su_news_components:
+    -
+      plugin: skip_on_empty
+      source: body
+      method: process
+    -
+      plugin: entity_revision_generate
+      entity_type: paragraph
+      bundle_key: type
+      bundle: stanford_wysiwyg
+      value_key: su_wysiwyg_text
+      ignore_case: true
+      values:
+        su_wysiwyg_text/value: body
+        su_wysiwyg_text/format: constants/basic_html
+  su_news_byline: su_spotlight_student_name
+  status:
+    plugin: default_value
+    default_value: 1
+  created:
+    plugin: callback
+    callable: strtotime
+  changed:
+    plugin: callback
+    callable: strtotime
+  langcode:
+    plugin: default_value
+    default_value: en
+destination:
+  plugin: 'entity:node'
+  default_bundle: stanford_news
+  overwrite_properties:
+    - title
```

Comment:
Remove this. Once the migration config is removed, editing can be done.

Reply:
@pookmish I've have a struggle with this. I tried leaving it out `overwrite_properties` or setting it to 0, `null` or `[]` or `{}`, and it makes all the fields uneditable. When I set it to Title, only the title fields is uneditable but at least the other fields are editable.

Reply:
This seems to allow for all the fields to be editable post-import: https://github.com/SU-SWS/earth_news_importer/blob/5a4216809e76137d60c81ad465051c9be5b8e6b2/config/install/migrate_plus.migration.earth_news_importer_2021.yml#L289C1-L289C43

Reply:
We don't need them to be editable. As i said, once we remove the migration config (via config delete or config import) the fields will become editable again.

Reply:
I see. Okay, I removed the overwrite_properties.